schema/me/consignments/create_submission_mutation.js

```diff
@@ -0,0 +1,24 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { SubmissionType } from "./submission"
+
+export default mutationWithClientMutationId({
+  name: "CreateSubmissionMutation",
+  description: "Create a new consigment using Convection",
+  inputFields: {
+    artist_id: {
```

Comment:
I think you should be able to create a submission with all of the input fields, not just the `artist_id`. Our clients only do the latter right now, but that's only because of the current (arbitrary) UI choice.